docs/docs/Get-Started/get-started-installation.md

```diff
@@ -120,53 +227,53 @@ A confirmation pane containing the selected version's changelog appears.
 3. To change to the selected version, click **Confirm**.
 The application restarts with the new version installed.
 
-## Common installation issues
+## Common OSS installation issues
```

Comment:
This should be a level-three heading and it should be located after "Install optional dependencies".